CommonMark/CommonMark.py

```diff
@@ -1078,8 +1078,7 @@ def closeUnmatchedBlocks(self, already_done, oldtip):
         if blank and container.last_line_blank:
             self.breakOutOfLists(container, line_number)
 
-        while container.t != "ExtensionBlock" and \
```

Comment:
I'd rather not undo this commit, which part of the test fails? (it shouldn't have impact)